.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -126,17 +138,84 @@ jobs:
           post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && ${{ matrix.luaengine && 'lualatex' || 'pdflatex' }} --shell-escape main_${{ matrix.language }}.tex
           root_file: main_${{ matrix.language }}.tex
 
+      - name: Check number of printable pages
+        id: check_printable_pages
+        if: ${{ matrix.language == 'en' }}
+        run: |
+          NUM_OF_PAGES=$(pdfinfo main_${{ matrix.language }}.pdf | grep Pages | grep -Po "\d+")
+          echo "NUM_OF_PRINTABLE_PAGES=${NUM_OF_PAGES}" >> "$GITHUB_OUTPUT"
+
+      - name: Check if printable page number matches
+        if: ${{ matrix.language == 'en' }}
+        run: |
+          PAGES_DIGITAL="${{ steps.check_pages.outputs.num_of_pages }}"
+          PAGES_PRINTABLE="${{ steps.check_printable_pages.outputs.num_of_printable_pages }}"
+          echo "Number of pages in digital:   ${PAGES_DIGITAL}"
+          echo "Number of pages in printable: ${PAGES_PRINTABLE}"
+          if [[ $((${PAGES_DIGITAL}+1)) != ${PAGES_PRINTABLE} ]]; then
+            echo "Pages in digital and printable versions differ by too much. See above. Please investigate this."
+            exit 1
```

Comment:
Let the artifact be downloadable even if the number of pages is wrong, otherwise it'll be very hard to debug why something overflown (on which page even); probably just move it after the upload

Reply:
Ok.